cli/src/main/kotlin/commands/NotifierCommand.kt

```diff
@@ -0,0 +1,80 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.cli.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.UsageError
+import com.github.ajalt.clikt.core.requireObject
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import org.ossreviewtoolkit.cli.GlobalOptions
+import org.ossreviewtoolkit.cli.utils.inputGroup
+import org.ossreviewtoolkit.cli.utils.readOrtResult
+import org.ossreviewtoolkit.notifier.Notifier
+import org.ossreviewtoolkit.utils.expandTilde
+import org.ossreviewtoolkit.utils.ortConfigDirectory
+
+class NotifierCommand : CliktCommand(name = "notify", help = "Create notifications based on an ORT result.") {
+    private val ortFile by option(
+        "--ort-file", "-i",
+        help = "The ORT result file to read as input."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+        .inputGroup()
+
+    private val notificationsFile by option(
+        "--notifications-file", "-n",
+        help = "The name of a kotlin script file containing notification rules."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .inputGroup()
+
+    private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
+
+    override fun run() {
+        val script = notificationsFile?.readText() ?: readDefaultNotificationsFile()
+
+        val ortResult = readOrtResult(ortFile)
+        val config = globalOptionsForSubcommands.config.notifier
+
+        val notifier = Notifier(ortResult, config)
+
+        notifier.run(script)
+    }
+
+    private fun readDefaultNotificationsFile(): String {
+        val notificationsFile = ortConfigDirectory.resolve("notifications.kts")
+
+        if (!notificationsFile.isFile) {
+            throw UsageError(
+                "No notification-file option specified and no default notification-file found at " +
```

Comment:
Please either use "--notification-file" or "notification file", but not something in between. I'd prefer "notification file" in both places to decouple from the concrete name of the option.

Reply:
I took this from another command. But I agree.